projects/packages/waf/src/class-waf-runner.php

```diff
@@ -16,19 +16,21 @@
  */
 class Waf_Runner {
 
-	const WAF_MODULE_NAME                     = 'waf';
-	const WAF_RULES_VERSION                   = '1.0.0';
-	const MODE_OPTION_NAME                    = 'jetpack_waf_mode';
-	const AUTOMATIC_RULES_ENABLED_OPTION_NAME = 'jetpack_waf_automatic_rules';
-	const IP_LISTS_ENABLED_OPTION_NAME        = 'jetpack_waf_ip_list';
-	const IP_ALLOW_LIST_OPTION_NAME           = 'jetpack_waf_ip_allow_list';
-	const IP_BLOCK_LIST_OPTION_NAME           = 'jetpack_waf_ip_block_list';
-	const RULES_FILE                          = __DIR__ . '/../rules/rules.php';
-	const ALLOW_IP_FILE                       = __DIR__ . '/../rules/allow-ip.php';
-	const BLOCK_IP_FILE                       = __DIR__ . '/../rules/block-ip.php';
-	const VERSION_OPTION_NAME                 = 'jetpack_waf_rules_version';
-	const RULE_LAST_UPDATED_OPTION_NAME       = 'jetpack_waf_last_updated_timestamp';
-	const SHARE_DATA_OPTION_NAME              = 'jetpack_waf_share_data';
+	const WAF_MODULE_NAME                          = 'waf';
+	const WAF_RULES_VERSION                        = '1.0.0';
+	const MODE_OPTION_NAME                         = 'jetpack_waf_mode';
+	const AUTOMATIC_RULES_ENABLED_OPTION_NAME      = 'jetpack_waf_automatic_rules';
+	const IP_LISTS_ENABLED_OPTION_NAME             = 'jetpack_waf_ip_list';
+	const IP_ALLOW_LIST_OPTION_NAME                = 'jetpack_waf_ip_allow_list';
+	const IP_BLOCK_LIST_OPTION_NAME                = 'jetpack_waf_ip_block_list';
+	const RULES_FILE                               = __DIR__ . '/../rules/rules.php';
```

Comment:
Maybe we should change the name? We have too many permutations of RULES_FILE naming and it's starting to get confused, maybe something like ENTRYPOINT_FILE?

Reply:
I've organized the constants a bit, and renamed them as follows via https://github.com/Automattic/jetpack/pull/28027/commits/70f04f4f7af1ad968d26e82106eddc07e0f320e0:
* `RULES_ENTRYPOINT_FILE`
* `AUTOMATIC_RULES_FILE`
* `IP_ALLOW_RULES_FILE`
* `IP_BLOCK_RULES_FILE`

This introduces a naming system where the sub-rule file are all named `FOO_BAR_RULES_FILE` while the main file is `RULES_ENTRYPOINT_FILE` for extra distinction.